news/post-conference-email.md

```diff
@@ -0,0 +1,80 @@
+title: Post-conference email
+date: 2016-09-24
+---
+This is an email that we are sending to all delegates after the conference
+
+* * *
+
+Hi everyone,
+
+Firstly, we'd like to thank you all for coming to the conference.  The committee all feel that it was a success, and we hope that you do too.
+
+
+### We want your feedback!
+
+To make next year's conference even better, we need you to tell us what worked and what didn't.  We'd love it if you could spend five minutes filling out [this short form](https://docs.google.com/forms/d/e/1FAIpQLSdzUtvUPSdINrGGWLBv9R3QuYYd-DLjRfKFU-GiVMD9V9NsCw/viewform).
+
+
+### Conference retrospective
+
+We're collecting the thoughts, views, and impressions from conference attendees on [our website](http://2016.pyconuk.org/news/20160920-impressions/).  There you can read what others have written, and you can add your own.
+
+
+### Talk slides
+
+We're also collecting links to talk slides, and will be adding them to the website.  If you gave a talk and would like us to link to your slides, please [email them to us](http://2016.pyconuk.org/contact/).
+
+
+### Code of Conduct report
+
+We've put together a [short report](http://2016.pyconuk.org/news/20160919-coc/) into how we handled two Code of Conduct issues.
+
+
+### Videos
+
+Thanks to [Autotrain](http://www.autotrain.org/), we were able to film most of the talks at this year's conference, and videos are being uploaded to our [YouTube channel](https://www.youtube.com/channel/UChA9XP_feY1-1oSy2L7acog) as fast as we can.
+
+
+### Lost property
+
+During the conference, the following items were handed in as lost property:
+
+* A red-lined grey Grin&Bear hoodie
+* A blue Fat Face zip-up hoodie
+* Tortoise-shell sunglasses
+
+If you think these are yours, [get in touch](http://2016.pyconuk.org/contact/).
+
+
+### Mailing list
+
+We're going to start a monthly mailing list to keep the UK Python community up to date with what's happening around the country.  We will add all conference attendees to the mailing list, unless you opt out.  Please [let us know](http://2016.pyconuk.org/contact/) by the end of September if you would like to opt out.
```

Comment:
What stops <new list> from suffering from the same issues as python-uk?  I know python-uk has its issues but I'm not keen to add yet another mailing list for people joining the community or ask existing community members to join a new list.

While the reply-all discussions are… well they exist, it would be nice to breath some new life into the list.  We could be the ones to drive a breath of fresh air/information rather than a new list.

[Relevant XKCD](https://xkcd.com/927/)